charts/leonardo/values.yaml

```diff
@@ -39,3 +39,22 @@ deployments:
     expose: true
     # deployments.standalone.serviceName -- Name of the default standalone Leonardo service
     serviceName: leonardo
+
+cronjob:
+  # deployments.standalone.name -- Name to use for the default standalone Leonardo deployment
+  name: resource-validator-cronjob
+  # deployments.standalone.replicas -- Number of replicas in the default standalone Leonardo deployment
+  replicas: 1
```

Comment:
under `cronjob`, remove the `name`, `replicas`, `expose`, `serviceName`, and `enabled` values, as they aren't used by the cronjob template